src/lib.rs

```diff
@@ -478,7 +478,7 @@ where
         <L::Service as Service<Request<Body>>>::Error: Into<Infallible> + 'static,
         <L::Service as Service<Request<Body>>>::Future: Send + 'static,
     {
-        self.routes = self.routes.clone().route_layer(service);
+        self.routes = self.routes.clone().layer(service);
```

Comment:
Used both methods for testing.

Reply:
Perhaps we should consider applying similar behavior to https://expressjs.com/en/guide/using-middleware.html#using-middleware. So we keep it familiar for express js users given that this crate aims to a similar approach